src/compas/scene/descriptors/colordict.py

```diff
@@ -54,6 +54,10 @@ def __get__(self, obj, otype=None):
             A defaultdict with the value stored in the default attribute corresponding to the descriptor as a default value.
 
         """
+        if obj is None:
+            # return the descriptor itself if accessed from the class
+            return self
+
```

Comment:
This was to enable the usage like `nodecolor=SceneObject.pointcolor` in a descendent class. When the `pointcolor` is accessed from class itself, we need `___get___` to return the descriptor itself instead of the data. 

But seems this mechanism is not needed for now after the adjustment. So I'm removing it.